owl/database/database.py

```diff
@@ -16,7 +18,9 @@ def __init__(self, config: DatabaseConfiguration):
         self.SessionLocal = scoped_session(self.session_factory)
 
     def init_db(self):
-        SQLModel.metadata.create_all(bind=self.engine)
```

Comment:
How does this work? Does it use the last created migration script to build the initial database as well?

Reply:
yes